src/adapter/src/coord/timeline.rs

```diff
@@ -138,12 +138,25 @@ impl<T: TimestampManipulation> TimestampOracle<T> {
 /// Interval used to persist durable timestamps. See [`DurableTimestampOracle`] for more
 /// details.
 pub static TIMESTAMP_PERSIST_INTERVAL: Lazy<mz_repr::Timestamp> = Lazy::new(|| {
-    Duration::from_secs(15)
+    Duration::from_secs(5)
         .as_millis()
         .try_into()
-        .expect("15 seconds can fit into `Timestamp`")
+        .expect("5 seconds can fit into `Timestamp`")
 });
 
+/// The Coordinator tries to prevent the persisted timestamp from exceeding
+/// a value [`TIMESTAMP_INTERVAL_UPPER_BOUND`] times [`TIMESTAMP_PERSIST_INTERVAL`]
+/// larger than the current system time.
+pub const TIMESTAMP_INTERVAL_UPPER_BOUND: u64 = 2;
```

Comment:
I settled on 2, because there's some tests that use a deterministic clock that need at least two intervals. Specifically this allows us to start a second instance immediately without blocking, if that second instance's clock happens to be behind the first instance.